src/com/theironyard/novauc/One.java

```diff
@@ -0,0 +1,19 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by Eric on 3/3/17.
+ */
+public class One {
+
+    public boolean sameFirstLast(int[] nums) {
+
+        if(nums.length >= 1 && nums[0] ==  nums[nums.length-1]){
+            return true; }
+        else {
+            return false;
+        }
```

Comment:
Need to write tests and resubmit. Refer to File I/o Continued lecture notes as a resource on how to write tests.  Coding Bat gives you test cases for most of the problems.